mapic/start_mapic.go

```diff
@@ -0,0 +1,20 @@
+package mistapiconnector
+
+import (
+	"github.com/livepeer/catalyst-api/config"
+)
+
+func NewMapic(cli *config.Cli) IMac {
```

Comment:
**nit:** Surprised these are the only configs needed by mist + that `start_mapic.go` doesn't start mapic but only creates a struct. Should this be moved to `mistapiconnector_app` as the constructor for the mapic component?